CHANGELOG.md

```diff
@@ -1,5 +1,45 @@
 # Release Notes
 
+## Version 1.10.0-beta.0 - 2020-04-09
+Changes:
+* fallback to alternate drivers on failure [#7389](https://github.com/kubernetes/minikube/pull/7389)
```

Comment:
There's a lot here: Do you mind splitting up the features & bugfixes like we've done in other releases?

Also, for PR's that introduce new flags, edit the PR name to mention or hand-edit it in the release notes.